README.md

```diff
@@ -839,12 +839,13 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * Relational Databases
     * [Django Models](https://docs.djangoproject.com/en/dev/topics/db/models/) - A part of Django.
     * [SQLAlchemy](https://www.sqlalchemy.org/) - The Python SQL Toolkit and Object Relational Mapper.
-        * [awesome-sqlalchemy](https://github.com/dahlia/awesome-sqlalchemy)
+    * [awesome-sqlalchemy](https://github.com/dahlia/awesome-sqlalchemy)
```

Comment:
Hello @twheys,
I believe you should keep an indentation here.

```suggestion
        * [awesome-sqlalchemy](https://github.com/dahlia/awesome-sqlalchemy)
```

Reply:
ah thanks for the feedback. overlooked this while fixing conflicts. it's fixed now.